src/os/bluestore/BlueStore.cc

```diff
@@ -4513,6 +4515,18 @@ bool BlueStore::test_mount_in_use()
   return ret;
 }
 
+void BlueStore::_check_kvbackend() {
+
+  auto kv_backend = cct->_conf->get_val<std::string>("bluestore_kvbackend");
+
+  if (kv_backend != bluestore_kvbackend) {
```

Comment:
i don't see where bluestore_kvbackend is initialized... this is called from the ctor, so won't this condition always be true?

Reply:
A good comeback. Thanks!